samples/Sentry.Samples.Android/MainActivity.cs

```diff
@@ -20,6 +20,11 @@ protected override void OnCreate(Bundle? savedInstanceState)
             // https://docs.sentry.io/platforms/android/configuration/
             // Enable Native Android SDK ANR detection
             options.Native.AnrEnabled = true;
+
+            options.SetBeforeSend(evt =>
+            {
+                return evt;
```

Comment:
If we're going to put this in the samples, could we make it a bit more interesting... e.g. inspecting the event and returning null for all events where event.x == y or something like that, to demonstrate how the beforesend callback can be used as a filter.

That way people are more likely to understand the purpose of the beforesend callback.